src/components/SingleReservation.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+
+export default class SingleResrvation extends Component {
+
+    updateStatus=()=>{
+        var status
+        if (this.props.single.status=== "free") {
+            status= "cancel"
+        }else if (this.props.single.status === "cancel") {
+            status= "free"
+        }
+        this.props.update(this.props.single,status)
+    }
+
+    render() {
+        var backgroundColor;
+        if (this.props.single.status === "free") {
+            backgroundColor=' btn btn-success'
+        }
+        if (this.props.single.status === "taken") {
+            backgroundColor='btn btn-danger'
+        }
+        if (this.props.single.status === "cancel") {
+            backgroundColor='btn btn-warning'
+        }
+        
+        return (
+            <tbody>  
```

Comment:
Taken Button was deleted when i clicked it:

![image](https://user-images.githubusercontent.com/48275618/76769543-53b72e00-67a5-11ea-9fce-7d751bf2b01f.png)

Reply:
The login button changed and the register seems not good:
![image](https://user-images.githubusercontent.com/48275618/76769669-6e89a280-67a5-11ea-9741-95b5daeb053e.png)